stdlib/strings/strings.flux

```diff
@@ -1,39 +1,780 @@
+// The strings package provides functions to manipulate UTF-8 encoded strings.
 package strings
 
 
-// Transformation functions
+// Transformation functions:
+//
+// The strings.title() function converts a string to title case.
+//
+// ## Parameters
+//
+// - `V` is the string value to convert.
+//
+// ## Convert all values of a column to title case
+//
+// ```
+//  import "strings"
+//
+//  data
+//      |> map(fn: (r) => ({ r with pageTitle: strings.title(v: r.pageTitle) }))
+//
 builtin title : (v: string) => string
+
+//
+// The strings.toUpper() function converts a string to uppercase.
```

Comment:
Same feedback here and for all of them below, start with the name of the builtin: `toUpper ...`